src/libraries/System.Private.Xml/src/System/Xml/Core/XmlTextEncoder.cs

```diff
@@ -520,19 +520,18 @@ internal void WriteEntityRef(string name)
         // for fragment of a string such as Write( string, offset, count).
```

Comment:
It seems like the justification for this method as outlined in the comment is no longer up to date. Could it all just be removed in favor of calling the `TextWriter.Writer(ROS<char>)` method?